bin/combinetf2_plot_hists_uncertainties.py

```diff
@@ -316,6 +316,12 @@ def make_plot(
             # unrolled ND histograms
             h_impact = hh.unrolledHist(h_impact, obs=[a.name for a in axes])
 
+        if len(h_impacts.axes) == 1:
```

Comment:
Would this cause problems in case there was a channel which really just had only one axis?

Reply:
Yes because the one axis is the impact axis, later a bin in this impact axis is selected which gives an object that only has "value" and "variance" instead of "values()" and "variances()" and the code would crash

Reply:
This happens for example for a histogram with only charge axis and then split plots by charge